cafy_pytest/plugin.py

```diff
@@ -515,49 +515,63 @@ def pytest_configure(config):
                 del os.environ["hybrid_mode"]
 
         #Debug Registration Server code
-        if CLS and int(CLS):
-            registration_id = os.environ.get("REG_ID", None)
-            CafyLog.registration_id = registration_id 
+        #Cant introduce switch case as this plugin is common for 3.6 and 3.11 env.
+        call_registeration = False
+        if CLS and not cafykit_debug_enable :
+            call_registeration = False
+            reg_id = os.environ.get("REG_ID", None)
+            CafyLog.logstash_server = LOGSTASH_SERVER
+            CafyLog.logstash_port = int(LOGSTASH_PORT)
```

Comment:
if LOGSTASH_PORT is not None: need to be checked

Reply:
The check added at the top to set CLS= 0 when the logstahs and etc is None.
These are env variables so they need to convert from string to typical type aka integer.

Reply:
Please re-open if the resolve is not supported.